apps/storefront-e2e/src/support/page_objects/product-details.page.ts

```diff
@@ -1,7 +1,10 @@
 import { TestProductData } from '../../types/product.type';
+import { ProductReferencesFragment } from '../page_fragments/product-references.fragment';
 import { QuantityInputFragment } from '../page_fragments/quantity-input.fragment';
 import { AbstractSFPage } from './abstract.page';
 
+const productReferencesFragment = new ProductReferencesFragment();
```

Comment:
Let's have `ProductReferencesFragment` as part of the PDP page, like

`getReferences = () => new ProductReferencesFragment()`

and then in tests

`pdp.getReferences().getWrapper()`

it will make tests code more readable, because you will always know on which page and with which page elements you do actions